### PR TITLE
Added User

### PR DESCRIPTION
**Pull request by Shaurya50211**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,6 +53,28 @@ <h4 class="mb-0"><strong>First Contribution Hacktoberfest 2022 List</strong></h4
...
+                <a href="https://github.com/" target="_blank"><i
+                    class="fa fa-github-square fa-2x icon-link icon-link-github" aria-hidden="true"></i></a>
+                <a href="https://www.linkedin.com/in/" target="_blank"><i
+                    class="fa fa-linkedin-square fa-2x icon-link icon-link-linkedin" aria-hidden="true"></i></a>
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+
```
Comment (ShehrozIrfan) [inline index.html:77]: Hey @Shaurya50211 !
Remove this extra line.

Comment (Shaurya50211) [reply index.html:77]: Sure

**Comment by Shaurya50211** [comment]
Removed line

**Comment by ShehrozIrfan** [comment]
@Shaurya50211 
You can follow me and star this repo for further updates.
Thanks. Have fun 👍

**Comment by Shaurya50211** [comment]
Thanks!

**Comment by ShehrozIrfan** [comment]
Hey there!
If your contribution is excluded for this repository, then please visit the repository [Ruby Rocks](https://github.com/ShehrozIrfan/ruby-rocks/) for contribution to hacktoberfest. It contains a plenty of beginner's friendly issues accepted for hacktoberfest.

Have fun!

**Pull request merged**